File: exercise01-B/src/program.py
#!/usr/bin/env python3

# Exercise 01-B: Fixed Totals

import sys


def main():
    while n := int(sys.stdin.readline()):
        print(sum(int(sys.stdin.readline()) for _ in range(n)))


if __name__ == "__main__":
    main()

# vim: set sts=4 sw=4 ts=8 expandtab ft=python:

File: exercise01-B/src/program.cpp
// Exercise 01-B: Fixed Totals

#include <iostream>
#include <istream>
#include <string>

// Prototypes

int SumSeries(std::istream& input, int n);

// Main Execution

int main(int argc, char* argv[]) {
    int n;
    while (std::cin >> n && n != 0)
        std::cout << SumSeries(std::cin, n) << '\n';

    return 0;
}

// Functions

int SumSeries(std::istream& input, int n) {
    int sum = 0;

    for (int i = 0; i < n; i++) {
        int j;
        input >> j;
        sum += j;
    }

    return sum;
}

// vim: set sts=4 sw=4 ts=8 ft=cpp:
